### ISSUE TITLE
Cells not running

### ISSUE THREAD

**Issue by mbpictures** [microsoft/AzureNotebooks#842]
While running a cell, the cell status becomes "[...]" but the cell doesn't actually run. It produces no output and nothing else happens. There is no other cell running

#### Which site did you access us from?
    [] https://notebooks.azure.com
    [x] https://studio.azureml.net

#### Describe the issue you are encountering
Running a cell labels the status with "[...]" but nothing happens.:
![image](https://user-images.githubusercontent.com/2772553/92640619-db59a180-f2dd-11ea-8d03-c20bdfe55c26.png)
Kernal is busy while nothing is active.

#### What steps can we take to reproduce this issue? If you have this in a notebook already, you can just share a link here.
Create a cell (code) enter (e.g.) !ls and click on run.

#### Which language/kernel were you using when you experienced this issue?
    [] Python 2.7 
    [] Python 3.5
    [x] Python 3.6 
    [] R 
    [] F#
    [] Terminal

#### What time (please include time zone) did this occur?
20.49 CEST

#### Which Web Browser(s) did you see the error in?
    [X] Chrome
    [] Firefox
    [] Internet Explorer / Edge (Please note this is not supported)
    [] Other (Please Specify)

#### Which Operating System are you using? Please specify the version as well if possible:
    [X] Windows 10
    [] MacOS
    [] Ubuntu
    [] Other (Please Specify)

**Comment by kkampf** [comment]
Hi @mbpictures, can you clarify where you were running the notebook and which notebook it was?

**Comment by mbpictures** [comment]
I'm running the notebook in the machine learning studio (ml.azure.com). The notebook was the TrainDatasetWithPyTorch.ipynb inside the root of my home folder.

I managed it to get it to work for now by creating a new compute instance. But I have another issue: sometimes the output of a cell has very high latency to display. e.g. I ran a cell and it took around 5min to display any output. The output it displayed was then for 5 commands. Commands used: rm, mkdir, cd, ...
